setup.cfg

```diff
@@ -66,6 +66,8 @@ doc =
 plotting =
     matplotlib>=3,<4
     pyvista>=0.29,<1
+rust =
+    gstools_core>=0.1.2,<1
```

Comment:
As above, I think `>=0.2.0,<1` is technically required.

Reply:
Even more important ;-)